core/ui/RootStylesheet.tid

```diff
@@ -4,12 +4,14 @@ code-body: yes
 \import [subfilter{$:/core/config/GlobalImportFilter}]
 \whitespace trim
 \rules only filteredtranscludeinline transcludeinline html
-<$let currentTiddler={{$:/language}} languageTitle={{!!name}}>
-	<$list filter="[all[shadows+tiddlers]!is[draft]tag[$:/tags/Stylesheet]]">
-		<$refresh-blocker refreshBlockerList={{{ [<currentTiddler>get[refresh-blocker-list]] }}}>
-			<style type="text/css">
-				<$view format={{{ [<currentTiddler>tag[$:/tags/Stylesheet/Static]then[text]else[plainwikified]] }}} mode="block"/>
-			</style>
-		</$refresh-blocker>
-	</$list>
-</$let>
+<$refresh-blocker refreshBlockerList="[all[tiddlers+shadows]is[draft]]">
```

Comment:
Shadow tiddlers can never be drafts. A draft tiddler is always part of the wiki-store.

Reply:
Oh you're right :slightly_smiling_face: